ci-operator/step-registry/telcov10n/sno-spoke/deploy/telcov10n-sno-spoke-deploy-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: telcov10n-sno-spoke-deploy
+  steps:
+    - ref: telcov10n-sno-spoke-gitea-deploy
+    - ref: telcov10n-sno-spoke-gitea-verify
+    - ref: telcov10n-sno-spoke-site-config-deploy
+    - ref: telcov10n-sno-spoke-site-config-verify
+    - ref: telcov10n-sno-spoke-gitops-setup
+    - ref: telcov10n-sno-spoke-gitops-verify
+    - ref: telcov10n-sno-spoke-cluster-install
+    - ref: telcov10n-sno-spoke-cluster-verify
+  documentation: |-
+    These chainned steps allow to both deploy and verify the k8s services into
```

Comment:
`These chained`
did you mean? - `verify the k8s services on the associated spoke cluster`?

Reply:
Same comment that above... note that the Spoke doesn't exist until it is deployed. All the operation are against the associated OCP Hub cluster.